runtime/doc/eval.txt

```diff
@@ -4573,15 +4573,16 @@ mkdir({name} [, {path} [, {prot}]])
 		If {prot} is given it is used to set the protection bits of
 		the new directory.  The default is 0755 (rwxr-xr-x: r/w for
 		the user readable for others).	Use 0700 to make it unreadable
-		for others.  This is only used for the last part of {name}.
-		Thus if you create /tmp/foo/bar then /tmp/foo will be created
-		with 0755.
+		for others.  Unlike vim is used for all parts of {name}.  Thus 
```

Comment:
We mark nvim differences with `{Nvim}` in the docs. This serves two purposes:

- makes it possible to generate a list of all differences by parsing the docs
- avoids noise phrases throughout the docs like "this is different than vi/vim..."

Suggestion:

```
the user readable for others).	Use 0700 to make it unreadable
for others.
                                                                         {Nvim}
                    {prot} is applied to all parts of {name}. Thus if you
                    create /tmp/foo/bar then /tmp/foo will be created with
                    0700.
```